xml/uefi.xml

```diff
@@ -264,54 +264,28 @@ Microsoft certificate for this. -->
-MOK list can only be modified in Secure Boot phase (before kernel is started).
-Enroll can also be done at boot time if key is available on EFI System
-Partition.
-</para>
--->
-   <para>
-    If the user (<quote>machine owner</quote>) wants to replace any components
-    of the boot process, Machine Owner Keys (MOKs) are to be used. The
-    <systemitem>mokutils</systemitem> tool will help with signing components
-    and managing MOKs.
-   </para>
-   <para>
-    The enrollment process begins with rebooting the machine and interrupting
-    the boot process (for example, pressing a key) when
-    <systemitem>shim</systemitem> loads. <systemitem>shim</systemitem> will
-    then go into enrollment mode, allowing the user to replace the default SUSE
-    key with keys from a file on the boot partition. If the user chooses to do
-    so, <systemitem>shim</systemitem> will then calculate a hash of that file
-    and put the result in a <quote>Boot Services Only</quote> variable. This
-    allows <systemitem>shim</systemitem> to detect any change of the file made
-    outside of Boot Services and thus avoid tampering with the list of
-    user-approved MOKs.
-   </para>
+   
    <para>
-    All of this happens during boot time&mdash;only verified code is executing
-    now. Therefore, only a user present at the console can use the machine
-    owner's set of keys. It cannot be malware or a hacker with remote access to
-    the OS because hackers or malware can only change the file, but not the
-    hash stored in the <quote>Boot Services Only</quote> variable.
+   To replace specific kernels, drivers, or other components that are part of
+   the boot process, you have to use Machine Owner Keys (MOKs). The
+   <systemitem>mokutil</systemitem> tool can help you to manage MOKs.
    </para>
    <para>
-    The boot loader, after having been loaded and verified by
-    <systemitem>shim</systemitem>, will call back to
-    <systemitem>shim</systemitem> when it wants to verify the kernel&mdash;to
-    avoid duplication of the verification code. <systemitem>Shim</systemitem>
-    will use the same list of MOKs for this and tell the boot loader whether it
-    can load the kernel.
+   You can create a MOK enrollment request with
+   <systemitem>mokutil</systemitem>. The request is stored in a UEFI runtime
+   (RT) variable called <systemitem>MokNew</systemitem>. During the next boot,
+   the <systemitem>shim</systemitem> bootloader detects
+   <systemitem>MokNew</systemitem> and loads <systemitem>MokManager</systemitem>
+   that presents yout with several options. You can use the <guimenu>Enroll key
+   from disk</guimenu> and <guimenu>Enroll hash from disk</guimenu> options to
+   add the key to the MokList. The <guimenu>Enroll MOK</guimenu> option can be
+   used to copy the key from the <systemitem>MokNew</systemitem> variable.
```

Comment:
```suggestion
   add the key to the MokList. Use the <guimenu>Enroll MOK</guimenu> option
   to copy the key from the <systemitem>MokNew</systemitem> variable.
```

Reply:
(Mainly because you were, very wisely, trying to eliminate the use of passive voice.)